Cargo.toml

```diff
@@ -47,6 +45,12 @@ strum = "0.8.0"
 strum_macros = "0.8.0"
 gdb-protocol = "0.1"
 burst = "0.0.2"
+rayon = "1.4"
+
+[dependencies.goblin]
+version = "0.2"
+default-features = false
+features = ["elf64", "elf32", "endian_fd", "std"]
```

Comment:
Why do you need elf32?

Reply:
Otherwise, it isn't possible to compile goblin...